src/openrct2/interface/Window.h

```diff
@@ -140,6 +140,16 @@ struct Widget
         moveDown(coords.y - top);
     }
 
+    void moveToX(int16_t x)
+    {
+        moveRight(x - left);
+    }
+
+    void moveToY(int16_t y)
+    {
+        moveDown(y - top);
+    }
+
```

Comment:
I think we should move our windows to store x/y pos and width/height internally. The current `MakeWidget` function was always intended to a transitional solution.

Fine for now, though.